applications/source/zeromq-source/pom.xml

```diff
@@ -3,30 +3,24 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <parent>
-        <groupId>org.springframework.cloud.stream.app</groupId>
-        <artifactId>stream-applications-core</artifactId>
-        <version>3.0.0-SNAPSHOT</version>
-        <relativePath/>
-    </parent>
-
     <artifactId>zeromq-source</artifactId>
-    <version>3.0.0-SNAPSHOT</version>
     <name>zeromq-source</name>
     <description>0MQ source apps</description>
     <packaging>jar</packaging>
 
-    <properties>
-        <zeromq.version>0.5.2</zeromq.version>
```

Comment:
So, this works now!

How that happened that we got a proper dependency tree?
Just those version changes gave us the way to go?

Reply:
As it turns out, I needed to add an entry to the `pom.xml` in `/functions/function-dependencies` for the new supplier. When I did that, it lined up all the dependency management from the functions when added to one of the application modules.